lib/KafkaBacklogMetrics.js

```diff
@@ -69,9 +69,14 @@ class KafkaBacklogMetrics extends EventEmitter {
     }
 
     isReady() {
-        return this._zookeeper &&
+        if (!(this._zookeeper &&
             this._zookeeper.getState().code ===
-            zookeeper.State.SYNC_CONNECTED.code;
+            zookeeper.State.SYNC_CONNECTED.code)) {
+            this._log.error('KafkaBacklogMetrics is not ready, zookeeper not connected',
+                { error: this._zookeeper.getState().code });
```

Comment:
I am not sure what else is part of the zookeeper state but it might be helpful to actually see why its failing and worth logging

Reply:
done 786501173242d922bd7bc8375d92b1c731b7495b